src/samples/p2p/js/peercall.js

```diff
@@ -0,0 +1,176 @@
+// Copyright (C) <2018-2020> Intel Corporation
```

Comment:
We usually just write the year we create this file. So there is no need to update it later.